### ISSUE TITLE
[ADE Support]:Clinical Reminders, Health Questionnaire Pilot - Notable COTS Accessibility Assessment iPhone with Voiceover

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#113004]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket to track accessibility assessment of COTS on iPhone with Voiceover._

## Description of the Work
The team is looking for an accessibility/useability audit of the software using AT. Please add any ally violations/usability issues you find in the main ticket, with a rating of how severe the issue is. No need to provide recommended fix. 

## Supporting Artifacts 
Below are the links to the Notable prototype along with log in credentials. They are active for 72 hours after the vendor send them to me. These expire on 6/28.

https://app.notablehealth.com/previsit/app/#/login/LX9OYDFQXXSL

DOB: 03/09/1945
Last Name: TWENTYSIX

Login is restricted to 1 user, and once experience is completed (by getting the final page), you can no longer renter the same experience. However, you can re-enter an experience to complete remaining questions.

## ADE Tasks
- [ ] Collaborate `task`
- [ ] ADE Support `task`
- [ ] Document `task`
- [x] Review `task`

## Acceptance Criteria
> [!TIP]
> Customize ADE Acceptance Criteria as needed. 
- [ ] Acceptance Criteria `Example 1`
- [ ] Acceptance Criteria `Example 2`
- [ ] Acceptance Criteria `Example 3`

### Ticket Updates
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`

**Comment by lakper** [comment]
Based on the findings above and recommendation, we will be closing this ticket and will not proceed with further testing for Notable on other devices/AT.
